### ISSUE TITLE
Yarn add failed

### ISSUE THREAD

**Issue by stijnvanderlaan** [wpk-cli/wpk#11]
```
Initialising new webpack configuration:
? Config filename: webpack.config.js
? Target execution environment: web
? Context directory path: ./
? Use multiple entries: No
? Entry file: index.js
? Output directory path: dist
? Bundle filename: bundle.js
? Add "~" alias for `context`: No
? Source mapping style: none
? Abort build on errors: never
? Use @babel/polyfill: No
? Add these plugins:
? Add these loaders: Babel, File, URL
? Create separate vendor bundle: No
? Install external dependencies: Yes
? Save answers for this command as a preset? No

> yarn add --dev @babel/preset-env babel-loader [object Object] [object Object]
```
This will fail with:
```
An unexpected error occurred: "https://registry.yarnpkg.com/[object: Not found
```

**Comment by sdgluck** [comment]
Thanks for the issue. Looks like a problem with how package names are derived from loader options. Will put in a fix asap!

**Comment by sdgluck** [comment]
@stijnvanderlaan Fix is in for this on latest (0.0.4). Thanks!
